src/smt.rs

```diff
@@ -393,6 +393,14 @@ pub fn literal_false() -> SMTExpr {
     literal_bool("false".to_string())
 }
 
+pub fn extract_msb_12_bytes(expr: impl Into<SMTExpr>) -> SMTExpr {
+    extract(255, 160, expr)
+}
+
+pub fn cleanup_msb_12_bytes(expr: impl Into<SMTExpr>) -> SMTExpr {
```

Comment:
`cleanup` is not really the right name because it does not perform the cleanup. It's more like `is_clean` or `is_address`?

Reply:
yea true